docs/outlook/privacy-and-security.md

```diff
@@ -137,11 +137,12 @@ Developers should follow the tiered permissions model to provide transparency an
   },
   ```
 
-- Developers can request the **restricted** permission if the Outlook add-in activates on a specific type of Outlook item (appointment or message), or on specific extracted entities (phone number, address, URL) being present in the item's subject or body.
+- Developers can request the **restricted** permission if the Outlook add-in activates on a specific type of Outlook item (appointment or message).
 
-  [!INCLUDE [outlook-contextual-add-ins-retirement](../includes/outlook-contextual-add-ins-retirement.md)]
-
-- Developers should request the **read item** permission if the Outlook add-in needs to read properties of the current item other than the default extracted entities, or write custom properties set by the add-in on the current item, but doesn't require reading or writing to other items, or creating or sending a message in the user's mailbox. For example, a developer should request **read item** permission if an Outlook add-in needs to look for an entity like a meeting suggestion, task suggestion, email address, or contact name in the item's subject or body, or uses a regular expression to activate.
+- Developers should request the **read item** permission if the Outlook add-in needs to:
+  - Read properties of the current item.
+  - Write custom properties set by the add-in on the current item, but doesn't require reading or writing to other items.
+  - Creating or sending a message in the user's mailbox.
```

Comment:
The lead-in doesn't make sense with this bullet i.e., "...the Outlook add-in needs to creating...". Consider rewriting.

```suggestion
  - Create or send a message in the user's mailbox.
```